molgenis-api-tests/pom.xml

```diff
@@ -9,6 +9,7 @@
         <version>5.3.0-SNAPSHOT</version>
     </parent>
     <artifactId>molgenis-api-tests</artifactId>
+    <description>API integration test test module.</description>
```

Comment:
The second "test" is because it is a "test module"? A bit of a personal preference issue, but I'd remove the second "test"